sshfs/spec.py

```diff
@@ -316,3 +316,12 @@ async def _execute(self, *args, **kwargs):
 
     def _open(self, path, *args, **kwargs):
         return SSHFile(self, path, *args, **kwargs)
+
+    @wrap_exceptions
+    async def _cat_file(self, path, **kwargs):
+        """Asynchronously fetch the contents of a file"""
+        async with self._pool.get() as channel:
+            async with channel.open(path, "rb") as f:
+                return await f.read()
+
+    cat_file = sync_wrapper(_cat_file)
```

Comment:
I think it's not needed. 

From the [docs](https://filesystem-spec.readthedocs.io/en/latest/async.html#implementing-new-backends):

>  These functions will either have sync versions automatically generated is the name is in the async_methods list, or can be directly created using sync_wrapper.

PTAL https://github.com/fsspec/filesystem_spec/blob/master/fsspec/asyn.py#L275 ... and https://github.com/fsspec/filesystem_spec/blob/master/fsspec/asyn.py#L913

Reply:
Aha, got it. I removed the sync_wrapper call.